packages/twenty-server/src/engine/core-modules/file/file-upload/services/file-upload.service.ts

```diff
@@ -32,6 +34,31 @@ export class FileUploadService {
     });
   }
 
+  private _sanitizeFile({
+    file,
+    ext,
+    mimeType,
+  }: {
+    file: Buffer | Uint8Array | string;
+    ext: string;
+    mimeType: string | undefined;
+  }): Buffer | Uint8Array | string {
+    const fileString = file.toString();
+
+    if (
```

Comment:
Did you check it couldn't be circumvented? Feels like it could be || more than && 
(e.g. maybe if wrong extension Chrome still attempts to display it so it's possible to circumvent sanitisation?)

Reply:
Agreed 👍